### ISSUE TITLE
How can I force Standard-Version to generate only changes for a version instead of take all commits for all versions?

### ISSUE THREAD

**Issue by fabienOuedraogo** [conventional-changelog/standard-version#757]
I generated a first version 1.0.0, then I generate a new version 1.0.1 with a new commit for a minor fix.
Standard version added into the version 1.0.1 the commit information for 1.0.0 and so on for each new version.

How can I force it to take only commits from the latest version?

Thank you for your help

**Comment by tannerhallman** [comment]
Sounds like https://github.com/conventional-changelog/standard-version/issues/126

Tl;dr sounds like it's unable to grab the correct tag from your git history and compare that with the new tag.

**Comment by fabienOuedraogo** [comment]
> Sounds like #126
> 
> Tl;dr sounds like it's unable to grab the correct tag from your git history and compare that with the new tag.

@tannerhallman  thank you for your response. I will check and clean my tags. Hope it will work.

**Comment by fabienOuedraogo** [comment]
@tannerhallman It seems the issue occurred because all my tags name doesn't start with `v` as semver specification. So after renamed tags, the changelog was generated correctly. 
Thank you for your response.
